psydac/api/ast/linalg.py

```diff
@@ -593,4 +642,86 @@ def transpose_3d( M:'float[:,:,:,:,:,:]', Mt:'float[:,:,:,:,:,:]', n1:"int64", n
     return
 
 _transpose  = {1:transpose_1d,2:transpose_2d, 3:transpose_3d}
-_args_dtype = {1:[repr('float[:,:]')]*2 + [repr('int64')]*11,2:[repr('float[:,:,:,:]')]*2 + [repr('int64')]*22, 3:[repr('float[:,:,:,:,:,:]')]*2 + ['int']*33}
+transpose_args_dtype = {1:[repr('float[:,:]')]*2 + [repr('int64')]*11,2:[repr('float[:,:,:,:]')]*2 + [repr('int64')]*22, 3:[repr('float[:,:,:,:,:,:]')]*2 + [repr('int64')]*33}
+#========================================================================================================
+def interface_transpose_1d( M:'float[:,:]', Mt:'float[:,:]', d_start:"int64", c_start:"int64", dim:"int64" ,n1:"int64", nc1:"int64", gp1:"int64", p1:"int64",
+                            nd1:"int64", ndT1:"int64", si1:"int64", sk1:"int64", sl1:"int64"):
```

Comment:
Given that this code is not automatically generated, can you please split these long lines of arguments at a width of 80-chars?

Reply:
done in [1d09218](https://github.com/pyccel/psydac/pull/160/commits/1d09218d700203883784eb8a5109e576b7f43b60)